config/opa-service/rego/common.rego

```diff
@@ -54,8 +54,11 @@ jwks_request(url) = http.send({
     "force_cache_duration_seconds": 3600
 })
 
-# Grab data from API
-data = http.send({"method": "get", "url": api_uri, "headers": {"accept": "text/rego", "fiware-service": request.tenant, "fiware-servicepath": request.service_path}}).body
+# Policies pull method
+# policies = http.send({"method": "get", "url": api_uri, "headers": {"accept": "text/rego", "fiware-service": request.tenant, "fiware-servicepath": request.service_path}}).body
```

Comment:
we should document how to enable set pull or push method, if possible the best solution would be to have an environment variable to pick one or the other method.

Reply:
This commented code was only left for the sake of keeping it "in mind". There's really no reason to use it when the push approach is better anyways. Maybe we can think of a different push approach later on.